tutorials/evoked/plot_10_evoked_overview.py

```diff
@@ -0,0 +1,313 @@
+# ``(n_channels, n_times)`` (in contrast to an :class:`~mne.Epochs` object,
+# which stores data of shape ``(n_epochs, n_channels, n_times)``). Thus to
+# create an :class:`~mne.Evoked` object, we'll start by epoching some raw data,
+# and then averaging together all the epochs from one condition:
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+events = mne.find_events(raw, stim_channel='STI 014')
+# we'll skip the "face" and "buttonpress" conditions, to save memory:
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4}
+epochs = mne.Epochs(raw, events, tmin=-0.3, tmax=0.7, event_id=event_dict,
+                    preload=True)
+evoked = epochs['auditory/left'].average()
+
+del raw  # reduce memory usage
+
+###############################################################################
+# Basic visualization of ``Evoked`` objects
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# We can visualize the average evoked response for left-auditory stimuli using
+# the :meth:`~mne.Evoked.plot` method, which yields a butterfly plot of each
+# channel type:
+
+evoked.plot()
+
+###############################################################################
+# Like the ``plot()`` methods for :meth:`Raw <mne.io.Raw.plot>` and
+# :meth:`Epochs <mne.Epochs.plot>` objects,
+# :meth:`evoked.plot() <mne.Evoked.plot>` has many parameters for customizing
+# the plot output, such as color-coding channel traces by scalp location, or
+# plotting the :term:`global field power <GFP>` alongside the channel traces.
+# See :ref:`tut-visualize-evoked` for more information about visualizing
+# :class:`~mne.Evoked` objects.
+#
+#
+# Subselecting ``Evoked`` data
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Unlike :class:`~mne.io.Raw` and :class:`~mne.Epochs` objects,
+# :class:`~mne.Evoked` objects do not support selection by square-bracket
+# indexing. Instead, data can be subselected by indexing the
+# :attr:`~mne.Evoked.data` attribute:
+
+print(evoked.data[:2, :3])  # first 2 channels, first 3 timepoints
+
+###############################################################################
+# .. note::
+#
+#   :class:`~mne.Evoked` objects use a :attr:`~mne.Evoked.data` *attribute*
+#   rather than a :meth:`~mne.Epochs.get_data` *method*; this reflects the fact
+#   that the data in :class:`~mne.Evoked` objects are always loaded into
+#   memory, never `memory-mapped`_ from their location on disk (because they
+#   are typically *much* smaller than :class:`~mne.io.Raw` or
+#   :class:`~mne.Epochs` objects).
+#
+#
+# Other useful attributes of :class:`~mne.Evoked` objects include
+# ``evoked.first`` and ``evoked.last`` (sample indices; where
+# sample ``0`` is the sample at time=0, so ``evoked.first`` may be negative).
```

Comment:
I hadn't used ``evoked.first`` and ``.last`` before, and this description got me a little confused, so I had to run your example and look at the values of these attributes to understand it. Now everything is super clear. So maybe I'm a slow thinker or bad reader, but I believe something like the following would have made things easier for me:

```suggestion
# ``evoked.first`` and ``evoked.last``. They correspond to *sample indices*,
# where sample ``0`` is the sample at time=0; so ``evoked.first`` may be
# negative. In our example, data were sampled at 600 Hz, and since the epochs
# start 0.3 sec before the experimental event, ``evoked.first`` is ``-180``.
```